icu_benchmarks/models/dl_models/tcn.py

```
@@ -0,0 +1,62 @@
+@gin.configurable
+class TemporalConvNet(DLPredictionWrapper):
+    """Temporal Convolutional Network. Adapted from TCN original paper https://github.com/locuslab/TCN"""
+
+    _supported_run_modes = [RunMode.classification, RunMode.regression]
+
+    def __init__(self, input_size, num_channels, num_classes, *args, max_seq_length=0, kernel_size=2, dropout=0.0, **kwargs):
+        super().__init__(
+            input_size=input_size,
+            num_channels=num_channels,
+            num_classes=num_classes,
+            *args,
```

Ruff

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)